cmd/pkiaas/routers.go

```diff
@@ -124,13 +124,6 @@ var routes = Routes{
 		api.GetCRLHandler,
 	},
 
-	Route{
-		"PurgeCRL",
-		strings.ToUpper("Post"),
-		"/pki/crl/purge",
-		api.PurgeCRLHandler,
-	},
-
```

Comment:
Why did we remove purge CRL?  Is this not necessary anymore?  If not, I need to update our Swagger schema.

Reply:
Yes we removed this endpoint because it will automatically be handled by the pki service internally. @rcobbins  this is correct right?

Reply:
Yeah, CRLs have a notoriously short TTL on their own... It doesn't make sense to have an endpoint to just purge them.  They will automatically ignore expired certificates when they are created.